Automattic/wp-calypso
# Odyssey Stats: Apply Jetpack colors

#### Proposed Changes

<img width="1262" alt="image" src="https://user-images.githubusercontent.com/4044428/210020448-26dc5ba5-de80-4d9d-899c-27b465c4c436.png">

Applies a Jetpack Green color palette to Odyssey Stats. These colors are not final and will undergo a design review in a follow-up PR.

#### Testing Instructions



* Check out this branch locally and build for your local Jetpack site (see Option 1 in PejTkB-3E-p2).
* Ensure that the new green color palette has been applied to Odyssey Stats.

#### Pre-merge Checklist
